NotifyNewProfile/handler.ts

```diff
@@ -0,0 +1,38 @@
+import { Context } from "@azure/functions";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { fromEither, left2v, taskEither, tryCatch } from "fp-ts/lib/TaskEither";
+import { Client as IDGCClient } from "../generated/dgc/client";
+import { errorsToError } from "../utils/conversions";
+
+const logPrefix = "NotifyNewProfile";
+
+export const NotifyNewProfile = (dgcClient: IDGCClient) => async (
```

Comment:
as we will dispatch the request to a different endpoint based on which fiscal code we receive, we must build client for every specific message

Reply:
Fixed in f0c089a